lib/pakyow/app/assets/console/styles/_data.scss

```diff
@@ -112,16 +112,20 @@
     background: $color-blue;
     position: absolute;
     color: $color-white;
-    margin-top: -37px;
+    margin-top: -36px;
     margin-left: 100px;
     border-top-right-radius: $radius;
     border-bottom-right-radius: $radius;
     @include rem(font-size, 14);
-    padding: 10px;
+    padding: 11px;
     padding-top: 11px;
     border-left: 1px solid darken($color-blue, 10%);
     cursor: pointer;
 
+    @include mobile {
```

Comment:
3 of 3 style changes that breaks the save actions. Reverting the first two and removing this font size change fixes it (was broken both on mobile and desktop).